modules/govuk_jenkins/templates/jobs/search_api_learn_to_rank.yaml.erb

```diff
@@ -0,0 +1,89 @@
+---
+- scm:
+    name: search-api
+    scm:
+      - git:
+          url: git@github.com:alphagov/search-api.git
+          branches:
+            - migrate-ltr-to-jenkins
+- job:
+    name: search-api-learn-to-rank
+    display-name: Search API Learn to Rank
+    project-type: freestyle
+    description: This job triggers Search API to generate training data, then use AWS SageMaker to train and deploy a model.
+    scm:
+      - search-api
+    logrotate:
+        numToKeep: 100
+    triggers:
+      - timed: |
+          TZ=Europe/London
+          0 22 * * SUN
+    properties:
+      - build-discarder:
+          days-to-keep: 30
+          artifact-num-to-keep: 5
+    publishers:
+      - trigger-parameterized-builds:
+          - project: Success_Passive_Check
+            condition: SUCCESS
+            predefined-parameters: |
+                NSCA_CHECK_DESCRIPTION=<%= @service_description %>
+                NSCA_OUTPUT=<%= @service_description %> success
+          - project: Failure_Passive_Check
+            condition: FAILED
+            predefined-parameters: |
+                NSCA_CHECK_DESCRIPTION=<%= @service_description %>
+                NSCA_OUTPUT=<%= @service_description %> failed
+    parameters:
+      - string:
+          name: AWS_ROLE_ARN
+          description: The ARN for the AWS Role to assume
+          default: <%= @aws_role_arn %>
+      - string:
+          name: ELASTICSEARCH_URI
+          description: The URI of Elasticsearch cluster for generating training data
+          default: <%= @elasticsearch_uri %>
+      - string:
+          name: IMAGE
+          description: The image name to use in SageMaker for training the model
+          default: <%= @image %>
+      - string:
+          name: TRAIN_INSTANCE_TYPE
+          description: The type of instance used to train the model
+          default: <%= @train_instance_type %>
+      - string:
+          name: TRAIN_INSTANCE_COUNT
+          description: The number instances used to train the model
+          default: <%= @train_instance_count %>
+      - string:
+          name: DEPLOY_INSTANCE_TYPE
+          description: The type of instance used to host the deployed model
+          default: <%= @deploy_instance_type %>
+      - string:
+          name: DEPLOY_INSTANCE_COUNT
+          description: The number instances used to host the deployed model
+          default: <%= @deploy_instance_count %>
+      - string:
+          name: GOVUK_ENVIRONMENT
+          default: <%= @environment %>
+    builders:
+      - shell: |
+          set -ex
+          chmod +x ./ltr/jenkins/start.sh
```

Comment:
Would it be possible to make the scripts in `./ltr/jenkins` executable for anyone who clones the repo, so you could remove this line? E.g. do `chmod +x ltr/jenkins/start.sh` and add that change to git? `git ls-files -s ltr/jenkins` should show you the perms changing from `100644` to `100755` (read/write to rw+executable).

Reply:
Ah struggling with this before, but now seems to have worked! Removed this line and updated the Search API branch with correct permissions.